backend/src/main/java/edu/netcracker/backend/dao/mapper/StatMapper.java

```diff
@@ -0,0 +1,31 @@
+package edu.netcracker.backend.dao.mapper;
+
+import edu.netcracker.backend.message.response.CarrierStatisticsResponse;
+import lombok.Setter;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.Date;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+@Setter
+public class StatMapper implements RowMapper<CarrierStatisticsResponse> {
+
```

Comment:
It's a good practice to check all the reference types on null. In 'if' statement add the braces